tests/trainer/optimization/test_parity_automatic_optimization.py

```diff
@@ -35,7 +37,11 @@
 
 class BaseParityAutomaticOptimizationModel(BoringModel):
 
-    def __init__(self, optimizer_cls, optimizer_is_mocked=False, accumulate_grad_batches=None):
+    def __init__(self,
+                 optimizer_cls,
+                 optimizer_is_mocked=False,
+                 accumulate_grad_batches=None,
+                 lr=0.1):
```

Comment:
```suggestion
    def __init__(
            self,
            optimizer_cls,
            optimizer_is_mocked=False,
            accumulate_grad_batches=None,
            lr=0.1,
    ):
```
if multi-line it shall be like this, but this still shall fit to one line